tests/run/pep448_extended_unpacking.pyx

```diff
@@ -272,6 +272,24 @@ def unpack_list_keep_originals(a, b, c):
     return [*a, *b, 2, *c]
 
 
+def t3938(x, l, m):
```

Comment:
I'd rather have an explanatory name here. Having to understand that this refers to a GH ticket, and then looking that up, is too much when I just want to know the purpose of this test.

Reply:
Sounds very reasonable, on it.

Reply:
Done